packages/contracts-bedrock/src/L1/SystemConfigInterop.sol

```diff
@@ -8,12 +8,31 @@ import { ERC20 } from "@openzeppelin/contracts/token/ERC20/ERC20.sol";
 import { SystemConfig } from "src/L1/SystemConfig.sol";
 import { ConfigType } from "src/L2/L1BlockInterop.sol";
 import { StaticConfig } from "src/libraries/StaticConfig.sol";
+import { Storage } from "src/libraries/Storage.sol";
 
 /// @title SystemConfigInterop
 /// @notice The SystemConfig contract is used to manage configuration of an Optimism network.
 ///         All configuration is stored on L1 and picked up by L2 as part of the derviation of
 ///         the L2 chain.
 contract SystemConfigInterop is SystemConfig {
+    /// @notice Storage slot where the address authorized to call `addDependency` is stored
+    bytes32 internal constant ADD_DEPENDENCY_ROLE_HOLDER_SLOT =
+        bytes32(uint256(keccak256("systemconfig.adddependencyrole")) - 1);
+
+    /// @notice Storage slot where the address authorized to call `removeDependency` is stored
+    bytes32 internal constant REMOVE_DEPENDENCY_ROLE_HOLDER_SLOT =
+        bytes32(uint256(keccak256("systemconfig.removedependencyrole")) - 1);
+
+    /// @notice Storage slot where the foundation multisig address is stored
+    bytes32 internal constant FOUNDATION_MULTISIG_SLOT =
```

Comment:
Lets make a single role called `dependencyManager` and have it be able to remove + add from the dependency set and that is it. We generally don't want to couple the idea of the foundation to the code since this is free software that anybody can operate in any way